src/content/docs/infrastructure/host-integrations/installation/secrets-management.mdx

```diff
@@ -743,3 +743,67 @@ variables:
 
 proxy: ${obfuscated_proxy}
 ```
+
+## Custom Commands [#custom-commands]
+
+<Callout variant="important">
+  The `command` configuration requires Infrastructure Agent `1.41.0` or greater.
+  Make sure the agent has the required permissions to execute the configured `command` based on your environment requirements.
+</Callout>
+
+You can load agent or integrations configurations from custom commands available in the same host.
+The Infrastructure Agent will detect when configurations should initially be loaded from a `command` output and keep values updated based on a `ttl` refresh time.
+
+The expected output format for the commmand is either JSON or String:
+
+### JSON
+The following JSON structure is expected as the main output (stdout) from the defined `command`: 
+```
+{
+  "data": {
+    "myKey": "myValue"
+  }, 
+  "ttl": "24h"
+}
+```
+- A top-level `data` field is required.
+- A top-level `ttl` field is optional, can be either included in the output from the command or defined with a fixed value in the configuration file.
+
+It can then be used in any configuration file using the `${myVariable.key}` notation.
+In the following example, a `my-custom-auth` command is executed with a `domain` parameter then loading the resulting `token` as an agent configuration: 
+
+```
+variables:
+  myToken:
+    command:
+      path: '/path/to/my-custom-auth-json'
+      args: ["domain", "myDomain"] 
```

Comment:
It's a bit confusing to me the `args` part as they look to me as "named parameters" instead of possitional ones. wdyt if we change it to something like:
 ```
args: ["--domain", "myDomain", "--other_param", "otherValue"]
```
or
 ```
args: ["arg1", "arg2"]
```
?